docs/explanations/security.md

```diff
@@ -1,32 +1,42 @@
 ---
 sidebar_label: "Security"
-title: ""
+title: "Epinio security"
+description: Epinio security, concepts and context
+keywords: [epinio, kubernetes, security, application development]
 ---
 
-## A Quick Note On Security
+<!--TODO:
+Should we be saying anything about source code level security, automated
+with Dependabot, Snyk, whatever?
+-->
```

Comment:
Oh. ... On one hand I see this as details the user has no need to know of.
On the other, doing so may be seen as an indicator of us taking the topic serious.
@olblak I suspect that this is more your level of things, i.e. beyond technical.

Reply:
OK, thanks, will merge for the moment.